package.json

```diff
@@ -7,9 +7,10 @@
     "build-storybook:components": "cd packages/components && yarn && yarn build-storybook",
     "build:components": "cd packages/components && yarn && yarn package",
     "build:playground": "cd packages/playground && yarn && yarn parcel-build",
-    "ci:lang": "yarn workspace @squiggle/lang ci",
-    "ci:components": "yarn ci:lang && yarn workspace @squiggle/components ci",
-    "ci:playground": "yarn ci:components && yarn workspace @squiggle/playground ci"
+    "ci:lang": "yarn workspace @quri/squiggle-lang ci",
```

Comment:
We removed the "ci" command in all of the packages, so this won't work. Nevertheless, these "build dependencies before building original package" is no longer needed anyway, so it's probably worth deleting all the ci commands here.